tests/appsec/test_automated_login_events.py

```diff
@@ -1163,6 +1163,8 @@ def validate_login_failure_headers(span):
 
 def assert_priority(span, meta):
```

Comment:
We need to check if this assert is needed and if it's correct. As far as I know (at least in java) set the manual.keep tag is not propagated to agent and when is set to true sampling priority is set to 2.